server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -497,6 +656,37 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 }
             }
             builder.endArray();
+            if (getDiscoveryNodesMetadata() != null) {
+                builder.startObject(UPLOADED_DISCOVERY_NODES_METADATA.getPreferredName());
+                getDiscoveryNodesMetadata().toXContent(builder, params);
+                builder.endObject();
+            }
+            if (getClusterBlocksMetadata() != null) {
+                builder.startObject(UPLOADED_CLUSTER_BLOCKS_METADATA.getPreferredName());
+                getClusterBlocksMetadata().toXContent(builder, params);
```

Comment:
this would change with XContent not used for its ser/de

Reply:
@shwetathareja This is only the UploadedMetadataAttribute object for ClusterBlocks and not ClusterBlocks itself. This will continue to use Xcontent